### DISCUSSION TITLE
Possibility to trigger manual update of the node information / status

### DISCUSSION THREAD

**Discussion by iridium2001** [dougiteixeira/proxmoxve#339] [Feature request] [answered]
The current API poll interval is 60 seconds, which is fine in respect to a modest production of data, but has the effect that it causes some delays for interacting with the VMs, when suspending or resuming them. 
It would be great if changes of the VM state would be timely updated. This could be done via a service / action on the integration side. 
If this was available, I could build it into the automation. 
Exposing the update interval itself would also be an option. Across the day, an update interval with even more than 60 seconds would also be fine.

**Comment by dougiteixeira** [comment, marked as answer]
Just use the action to update, by setting up an automation you can work with the update rate you want.

```
action: homeassistant.update_entity
data:
  entity_id:
    - binary_sensor.node_pve_status
```

**Comment by mZ738** [comment]
Douglas' answer still holds — `homeassistant.update_entity` refreshes a coordinator on demand, so an automation can poll a guest right after a start or suspend. What is new in 5.2.0 is a **polling interval** option down to 30 seconds, which takes the edge off the delay without an automation. https://github.com/dougiteixeira/proxmoxve/releases/tag/5.2.0
